History.md

```diff
@@ -1,4 +1,17 @@
 
+v1.0.0 / 2020-09-17
+==================
+
+* pkg: add standard
+* pkg: add release script
+* add package-lock.json
+* Npmify (#9)
+* Merge pull request #8 from voliva/npmrc-scopes
+* Support .npmrc scopes
+* update copyright
+* sponsors
```

Comment:
why did you update the changelog? that's my responsibility as a maintainer. if you look closely, the format changed, and the version you added is an outdated one

Reply:
To be honest, I don't know why I did that. As you said, there isn't much reason to do this. I will revert it, sorry

Reply:
I reverted this change.